source/assets/css/regions/_banner.scss

```diff
@@ -40,4 +40,11 @@
       border: 0;
     }
   }
-}
\ No newline at end of file
+}
+
+#docsearch {
```

Comment:
This is a nitpick, but I usually like to separate out styles that are specific to third-party stuff into it's own partial. Especially when it relies on IDs to target.

Reply:
Sure! Where would you like it to go?

Reply:
how about `source/assets/css/vendor/_docsearch.scss`?

Reply:
Made my best guess. Feel free to move it around after this lands.

Reply:
did you not like my suggestion? (no worries. can move it later) :)